src/99-appendices/06-meg-file-formats.md

```diff
@@ -180,10 +180,19 @@ Where:
 
 ```Text
 sub-control01_ses-001_task-rest_run-01_meg/
-    sub-control01_ses-001_task-rest_run-01_markers.<mrk,sqd>
+    sub-control01_ses-001_task-rest_run-01[_acq-<acq-label>]_markers.<mrk,sqd>
```

Comment:
here and elsewhere, should be `_acq-<label>` as per my unification through the entire specification a little back in #93